### ISSUE TITLE
Save keras training history

### ISSUE THREAD

**Issue by epa095** [equinor/gordo-components#138]
Keras models have a history field which contains the loss/acc for all the epochs, but it does not get saved by model.save. I propose that we, in `save_to_dir` and `load_from_dir` handle the history field as well if it is present (probably only present if `fit` has been called).
